package.json

```diff
@@ -26,7 +26,7 @@
     "format-check": "npx prettier --check \"base/**/*.ts\"",
     "format": "npx prettier --write \"base/**/*.ts\"",
     "docs": "node scripts/generate_docs.js",
-    "postinstall": "npx husky install && npm dedupe",
+    "_postinstall": "npx husky install && npm dedupe",
```

Comment:
please restore this change, I don't know why this happens. But restoring is fine. This happens only in the first you setup the repo.